.dockerignore

```diff
@@ -2,6 +2,10 @@
 # Ignore everything
 **
 
+!samples/brokers/zeroconf
```

Comment:
if we integrate zeroconf as an "inbox" protocol (update root Cargo.toml, use our build workflow actions), we shouldn't need to do this (the existing Rust code for agent, controller, & brokers is not built with a docker build command).